workflows/computational-chemistry/fragment-based-docking-scoring/.dockstore.yml

```diff
@@ -1,7 +1,13 @@
 version: 1.2
 workflows:
 - name: main
-  primaryDescriptorPath: /fragment-based-docking-scoring.ga
   subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /fragment-based-docking-scoring.ga
   testParameterFiles:
   - /fragment-based-docking-scoring-tests.yml
+  authors:
+  - name: Simon Bray
+    identifier: 0000-0002-0621-6705
```

Comment:
Any idea why it is indentifier and not orcid?

Reply:
Maybe 
https://github.com/galaxyproject/planemo/blob/2c71d887e7b38ae9124ce0fa6a70e8f82bf77826/planemo/workflow_lint.py#L99-L104

Reply:
On my computer I get:
```
>>> orcid = re.findall(r"(?:\d{4}-){3}\d{4}", "https://orcid.org/0000-0001-6879-5194")
>>> orcid
['0000-0001-6879-5194']
>>> import requests
>>> requests.get(
...                                     f"https://orcid.org/{orcid[0]}",
...                                     headers={"Accept": "application/xml"},
...                                     allow_redirects=False,
...                                 ).status_code
302

```

Reply:
I think `allow_redirects=False` is the problem here, do you remember why we did that ?

Reply:
(302 is a redirect)

Reply:
Because if we don't an erroneous orcid redirect to login page...

Reply:
Then we don't know if it is valid or not... But this was working before I wrote a test on planemo that passed at one point.

Reply:
That doesn't seem to be the case anymore:

```
In [3]: import requests

In [4]: requests.get("https://orcid.org/0000-0001-6879-5193", allow_redirects=True, headers={"Accept": "application/xml"}).status_code
Out[4]: 404

In [5]: requests.get("https://orcid.org/0000-0001-6879-5194", allow_redirects=True, headers={"Accept": "application/xml"}).status_code
Out[5]: 200
```

Reply:
Arrgh sorry. Then we need to modify planemo...